.gitignore

```diff
@@ -17,4 +17,5 @@ Source/.vs/
 Source/.vscode/
 .vscode/
 .idea/*
-.DS_Store
\ No newline at end of file
+.DS_Store
+.cache
```

Comment:
Out of curiosity, what is `.cache`?

And, as always, these unrelated changes should at the very least have their own commit and description. Perhaps it can even have a dedicated PR to avoid getting the change blocked by other potentially controversial changes.